tests/integration/targets/shutdown/tasks/main.yml

```
@@ -69,3 +76,9 @@
       - '"-d 100" in shutdown_result["shutdown_command"]'
       - '"-d 0" in shutdown_result_minus["shutdown_command"]'
   when: ansible_system == 'VMKernel'
+
+- name: Remove systemd_sysv in ubuntu 18 in case it has been installed in test
+  apt:
+    name: systemd-sysv
+    state: absent
+  when: systemd_sysv_install is changed and ansible_distribution == 'Ubuntu' and ansible_distribution_major_version is version('18', '>=')
```

```suggestion
  when: systemd_sysv_install is changed
```
This should also work (since skipped => not changed), and make it easier to change the conditions when this package should be installed (you don't have to adjust the condition here as well).

---

Sure. Fixed in https://github.com/ansible-collections/community.general/pull/700/commits/9692bb42680e74ae795fd275a8feaf956c354948.